clang/test/SemaSYCL/CudaCompat/delayed-diags.cu

```diff
@@ -0,0 +1,37 @@
+// RUN: %clang_cc1 %s -fsycl-is-host -fsycl-cuda-compatibility \
+// RUN:   -internal-isystem %S/../../SemaCUDA/Inputs \
+// RUN:   -internal-isystem %S/../Inputs \
+// RUN:   -fsycl-targets=nvptx64-nvidia-cuda -triple x86_64-unknown-linux \
+// RUN:   -emit-llvm -o - -verify -verify-ignore-unexpected=note
```

Comment:
Why `-verify-ignore-unexpected=note`

Reply:
I took the cmd from another and missed this. Removed.